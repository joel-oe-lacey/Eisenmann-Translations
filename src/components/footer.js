import React from "react"
import {
  graphql,
  StaticQuery
} from "gatsby"
import Image from "gatsby-image"
import styled from 'styled-components'
// import { makeStyles, styled } from '@material-ui/core/styles';
import List from '@material-ui/core/List';
import Divider from '@material-ui/core/Divider';
import ListItem from '@material-ui/core/ListItem';
import ListItemText from '@material-ui/core/ListItemText';
import ListSubheader from '@material-ui/core/ListSubheader';
import Link from './link';
import TableFooter from '@material-ui/core/TableFooter';

const StyledFooter = styled(TableFooter)({
    height: 'min-height',
    width: '100%',
    backgroundColor: 'grey',
    display: 'flex',
    flexDirection: 'row',
    justifyContent: 'flex-end'
})

const FetchFooter = ({ data }) => { 
  const postsByCategory = data.allMarkdownRemark.edges.reduce((groupPosts, { node }) => {
    const category = node.frontmatter.category ? node.frontmatter.category : 'none';

    if (!groupPosts[category]) {
      groupPosts[category] = [node]
    } else {
      groupPosts[category].push(node)
    }

    return groupPosts;
  }, {})

  const list = () => (
    <React.Fragment>
        {
          Object.keys(postsByCategory).map(category => {
            return (
              <React.Fragment key={category}>
                <List>
                  <ListSubheader>{category}</ListSubheader>
                  {
                    postsByCategory[category].map(node => {
                      const slug = node.fields.slug;
                      const title = node.frontmatter.title;
                      
                      return (
                      <ListItem button key={title}>
                        <Link to={slug}>
                          <ListItemText primary={title} />
                        </Link>
                      </ListItem>
                      )
                    })
                  }
                </List>
              </React.Fragment>
            )
          })
        }
    </React.Fragment>
  );

  return (
    <StyledFooter>
        {list()}
    </StyledFooter>
  )
}

const Footer = () => {
    return (
    <StaticQuery
      query={graphql`
        query {
          site {
            siteMetadata {
              title
            }
          }
          allMarkdownRemark {
            edges {
              node {
                excerpt
                fields {
                  slug
                }
                frontmatter {
                  title
                  type
                  category
                }
              }
            }
          }
          avatar: file(absolutePath: { regex: "/Banner_Ende.png/" }) {
            childImageSharp {
              fixed(width: 960, height: 100) {
                ...GatsbyImageSharpFixed
              }
            }
          }
        }
      `}
      render={data => <FetchFooter data={data} />}
    />
  )
}

export default Footer