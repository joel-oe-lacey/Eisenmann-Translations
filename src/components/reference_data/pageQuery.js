export const queriedMenu = {
  "site": {
    "siteMetadata": {
      "title": "Eisenmann Translations"
    }
  },
  "allMarkdownRemark": {
    "edges": [
      {
        "node": {
          "excerpt": "Impressum von fachuebersetzungen.de Wir freuen uns auf Ihre Anfrage - und noch mehr über Ihre Übersetzungsaufträge! Wir unterbreiten Ihnen…",
          "fields": {
            "slug": "/about/contact-us/de/"
          },
          "frontmatter": {
            "title": "Kontakt",
            "type": "pages",
            "category": "Kontakt",
            "redirectLink": null,
            "locale": "de"
          }
        }
      },
      {
        "node": {
          "excerpt": "Legal We look forward to hearing from you, and even more to receiving your translations! We would be happy to make you an offer for a…",
          "fields": {
            "slug": "/about/contact-us/en/"
          },
          "frontmatter": {
            "title": "Contact Us",
            "type": "pages",
            "category": "Contact",
            "redirectLink": null,
            "locale": "en"
          }
        }
      },
      {
        "node": {
          "excerpt": "Praktikant/in der Übersetzungswissenschaften Übersetzer-Praktikum im sprachlichen Bereich Anforderungen an Übersetzer-Studenten…",
          "fields": {
            "slug": "/about/jobs/de/"
          },
          "frontmatter": {
            "title": "Jobs",
            "type": "pages",
            "category": "Kontakt",
            "redirectLink": null,
            "locale": "de"
          }
        }
      },
      {
        "node": {
          "excerpt": "We’re Looking For… Intern Linguistic intern for 2-6 months Requirements: must be enrolled in, or have graduated from, a German, British…",
          "fields": {
            "slug": "/about/jobs/en/"
          },
          "frontmatter": {
            "title": "Jobs",
            "type": "pages",
            "category": "Contact",
            "redirectLink": null,
            "locale": "en"
          }
        }
      },
      {
        "node": {
          "excerpt": "Native Speakers The mother tongue principle is the rule for all translations. It states that a translator shall only translate into his/her…",
          "fields": {
            "slug": "/about/native-speakers/"
          },
          "frontmatter": {
            "title": "Native Speakers",
            "type": "pages",
            "category": "About",
            "redirectLink": null,
            "locale": "en"
          }
        }
      },
      {
        "node": {
          "excerpt": "Muttersprachler-Übersetzungen Das Muttersprachlerprinzip ist die Regel für alle Übersetzungen. Es besagt, daß der Übersetzer stets in die…",
          "fields": {
            "slug": "/about/native-speakers/german/"
          },
          "frontmatter": {
            "title": "Muttersprachler-Übersetzungen",
            "type": "pages",
            "category": "Über",
            "redirectLink": null,
            "locale": "de"
          }
        }
      },
      {
        "node": {
          "excerpt": "Prices Translations are usually (with a few exceptions) priced based on the amount of text translated in normal lines. Normal lines of 5…",
          "fields": {
            "slug": "/about/prices/"
          },
          "frontmatter": {
            "title": "Prices",
            "type": "pages",
            "category": "Contact",
            "redirectLink": null,
            "locale": "en"
          }
        }
      },
      {
        "node": {
          "excerpt": "Zeilenpreise unserer Übersetzungen Von einigen wenigen Ausnahmen abgesehen werden Übersetzungen generell nach der übersetzten Textmenge…",
          "fields": {
            "slug": "/about/prices/german/"
          },
          "frontmatter": {
            "title": "Preise",
            "type": "pages",
            "category": "Kontakt",
            "redirectLink": null,
            "locale": "de"
          }
        }
      },
      {
        "node": {
          "excerpt": "Recht, Wirtschaft, Medizin, Technik, EDV… Wir haben langjährige Erfahrungen (seit 1990!) in der Übersetzung folgender Fachgebiete und setzen…",
          "fields": {
            "slug": "/about/subject-areas/german/"
          },
          "frontmatter": {
            "title": "Fachgebiete",
            "type": "pages",
            "category": "Über",
            "redirectLink": null,
            "locale": "de"
          }
        }
      },
      {
        "node": {
          "excerpt": "Law, Economics, Medicine, Technology, IT… We have many years of experience (we started in 1990!) translating in the following subject areas…",
          "fields": {
            "slug": "/about/subject-areas/"
          },
          "frontmatter": {
            "title": "Subject Areas",
            "type": "pages",
            "category": "About",
            "redirectLink": null,
            "locale": "en"
          }
        }
      },
      {
        "node": {
          "excerpt": "",
          "fields": {
            "slug": "/about/redirect/lang-de/"
          },
          "frontmatter": {
            "title": "Sprache",
            "type": "pages",
            "category": "Über",
            "redirectLink": "/lang/german/",
            "locale": "de"
          }
        }
      },
      {
        "node": {
          "excerpt": "A redirect example file. Here content would not be displayed if the redirectLink frontmatter property is present a link will be created that…",
          "fields": {
            "slug": "/about/redirect/lang-en/"
          },
          "frontmatter": {
            "title": "Language",
            "type": "pages",
            "category": "About",
            "redirectLink": "/lang/english/",
            "locale": "en"
          }
        }
      }
    ]
  },
  "banner": {
    "childImageSharp": {
      "fixed": {
        "base64": "data:image/jpeg;base64,/9j/2wBDABALDA4MChAODQ4SERATGCgaGBYWGDEjJR0oOjM9PDkzODdASFxOQERXRTc4UG1RV19iZ2hnPk1xeXBkeFxlZ2P/2wBDARESEhgVGC8aGi9jQjhCY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2P/wgARCAABABQDASIAAhEBAxEB/8QAFQABAQAAAAAAAAAAAAAAAAAAAAP/xAAUAQEAAAAAAAAAAAAAAAAAAAAA/9oADAMBAAIQAxAAAAGYAP/EABYQAQEBAAAAAAAAAAAAAAAAAEEAAf/aAAgBAQABBQLZP//EABQRAQAAAAAAAAAAAAAAAAAAABD/2gAIAQMBAT8BP//EABQRAQAAAAAAAAAAAAAAAAAAABD/2gAIAQIBAT8BP//EABcQAQADAAAAAAAAAAAAAAAAAAABAoH/2gAIAQEABj8CsxD/xAAYEAEAAwEAAAAAAAAAAAAAAAAAMUFxsf/aAAgBAQABPyGWFng//9oADAMBAAIAAwAAABADz//EABQRAQAAAAAAAAAAAAAAAAAAABD/2gAIAQMBAT8QP//EABQRAQAAAAAAAAAAAAAAAAAAABD/2gAIAQIBAT8QP//EABYQAQEBAAAAAAAAAAAAAAAAAAABof/aAAgBAQABPxDIUIH/2Q==",
        "width": 960,
        "height": 50,
        "src": "/static/0771bdd4afaa23b4519b932f5bbb65bf/1454c/Banner_Unten.jpg",
        "srcSet": "/static/0771bdd4afaa23b4519b932f5bbb65bf/1454c/Banner_Unten.jpg 1x"
      }
    }
  },
  "certification": {
    "childImageSharp": {
      "fixed": {
        "base64": "data:image/jpeg;base64,/9j/2wBDABALDA4MChAODQ4SERATGCgaGBYWGDEjJR0oOjM9PDkzODdASFxOQERXRTc4UG1RV19iZ2hnPk1xeXBkeFxlZ2P/2wBDARESEhgVGC8aGi9jQjhCY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2P/wgARCAAdABQDASIAAhEBAxEB/8QAGQAAAgMBAAAAAAAAAAAAAAAAAAMBAgUE/8QAFgEBAQEAAAAAAAAAAAAAAAAAAgQD/9oADAMBAAIQAxAAAAFy1Vpn1xRPvz014ZSOM1//xAAgEAABAwIHAAAAAAAAAAAAAAACAAEDBBQQEhMhIjEz/9oACAEBAAEFAjqiYrs1EbygTDm4ql8jpSc7QlCGmD74dr//xAAaEQACAgMAAAAAAAAAAAAAAAAAAQIQERIx/9oACAEDAQE/Acsjw0Vf/8QAGxEAAgEFAAAAAAAAAAAAAAAAAAIBEhMhMVH/2gAIAQIBAT8BpXg2y4xOT//EAB0QAAIBBAMAAAAAAAAAAAAAAAABEgIRITEgMoH/2gAIAQEABj8CassM0iTK77kx6PRuSyzsiL4f/8QAHxABAAIBBAMBAAAAAAAAAAAAAQARITFBUXEQYdHx/9oACAEBAAE/ITvWhvAvsysgN1iNxPYlHDLva5x8pqOB0Z+KxDQW7xMl14XpP//aAAwDAQACAAMAAAAQJ8AP/8QAFxEBAQEBAAAAAAAAAAAAAAAAAQARMf/aAAgBAwEBPxB37LQsvBhhf//EABYRAQEBAAAAAAAAAAAAAAAAAAEAEf/aAAgBAgEBPxAwhxhBS1rf/8QAIBABAQACAQMFAAAAAAAAAAAAAREAIVFBYdFxgaHB8P/aAAgBAQABPxCf0hELGc4sNZz5MOXJFzWKjGlZ1Tv64r270NHkmQmIaz3wliADrreM/V/WV4pQQi98RpDo6G/GEBIb7HjCRCAkz//Z",
        "width": 100,
        "height": 145,
        "src": "/static/00f598e2bfece3cac12348633027a854/1b3ab/BDUE-Mitglied-Logo.jpg",
        "srcSet": "/static/00f598e2bfece3cac12348633027a854/1b3ab/BDUE-Mitglied-Logo.jpg 1x"
      }
    }
  }
}

export const queriedHeader = {
  "background": {
    "childImageSharp": {
      "fluid": {
        "base64": "data:image/jpeg;base64,/9j/2wBDABALDA4MChAODQ4SERATGCgaGBYWGDEjJR0oOjM9PDkzODdASFxOQERXRTc4UG1RV19iZ2hnPk1xeXBkeFxlZ2P/2wBDARESEhgVGC8aGi9jQjhCY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2P/wgARCAANABQDASIAAhEBAxEB/8QAGAAAAwEBAAAAAAAAAAAAAAAAAAQFAQP/xAAUAQEAAAAAAAAAAAAAAAAAAAAA/9oADAMBAAIQAxAAAAGnivEeJgf/xAAbEAACAgMBAAAAAAAAAAAAAAAAAgEDBBESFP/aAAgBAQABBQKciuD01E5AzbOVkatd/wD/xAAUEQEAAAAAAAAAAAAAAAAAAAAQ/9oACAEDAQE/AT//xAAUEQEAAAAAAAAAAAAAAAAAAAAQ/9oACAECAQE/AT//xAAbEAABBAMAAAAAAAAAAAAAAAAAASExMgIRgf/aAAgBAQAGPwJ8iw2uqQhVCD//xAAeEAACAgAHAAAAAAAAAAAAAAAAAREhQVFhcYGRwf/aAAgBAQABPyFtEGWp0BE1ZbJIthK8jDy4P//aAAwDAQACAAMAAAAQc8//xAAUEQEAAAAAAAAAAAAAAAAAAAAQ/9oACAEDAQE/ED//xAAUEQEAAAAAAAAAAAAAAAAAAAAQ/9oACAECAQE/ED//xAAbEAEAAwEBAQEAAAAAAAAAAAABABEhMUFxkf/aAAgBAQABPxCjm3VcesSGktFA3eR7qvoK/kD9sCOjq7jYyxm4R//Z",
        "aspectRatio": 1.5,
        "src": "/static/ebfd1f9a48e20ddbaa322313fb9b00b9/c4ecb/river.jpg",
        "srcSet": "/static/ebfd1f9a48e20ddbaa322313fb9b00b9/1d4ac/river.jpg 1125w,\n/static/ebfd1f9a48e20ddbaa322313fb9b00b9/c20da/river.jpg 2250w,\n/static/ebfd1f9a48e20ddbaa322313fb9b00b9/c4ecb/river.jpg 4500w,\n/static/ebfd1f9a48e20ddbaa322313fb9b00b9/61a1d/river.jpg 4594w",
        "sizes": "(max-width: 4500px) 100vw, 4500px"
      }
    }
  }
}