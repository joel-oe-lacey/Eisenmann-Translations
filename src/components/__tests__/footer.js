import React from "react"
import renderer from "react-test-renderer"
import {FetchFooter as Footer} from "../footer"
import { queriedMenu } from '../reference_data/pageQuery'

describe("Footer", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<Footer 
          data={queriedMenu}
        />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})