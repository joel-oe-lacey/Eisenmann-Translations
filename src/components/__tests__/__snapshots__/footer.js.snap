// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
<footer
  className="sc-bdnylx inwbrT"
>
  <ul
    className="MuiList-root MuiList-padding"
  >
    <li
      className="MuiListSubheader-root MuiListSubheader-sticky MuiListSubheader-gutters"
    >
      Contact
    </li>
    <a
      href="/about/contact-us/en/"
    >
      <div
        aria-disabled={false}
        className="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
        onBlur={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        role="button"
        tabIndex={0}
      >
        <div
          className="MuiListItemText-root"
        >
          <span
            className="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
          >
            Contact Us
          </span>
        </div>
      </div>
    </a>
    <a
      href="/about/jobs/en/"
    >
      <div
        aria-disabled={false}
        className="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
        onBlur={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        role="button"
        tabIndex={0}
      >
        <div
          className="MuiListItemText-root"
        >
          <span
            className="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
          >
            Jobs
          </span>
        </div>
      </div>
    </a>
    <a
      href="/about/prices/"
    >
      <div
        aria-disabled={false}
        className="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
        onBlur={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        role="button"
        tabIndex={0}
      >
        <div
          className="MuiListItemText-root"
        >
          <span
            className="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
          >
            Prices
          </span>
        </div>
      </div>
    </a>
  </ul>
  <ul
    className="MuiList-root MuiList-padding"
  >
    <li
      className="MuiListSubheader-root MuiListSubheader-sticky MuiListSubheader-gutters"
    >
      About
    </li>
    <a
      href="/about/native-speakers/"
    >
      <div
        aria-disabled={false}
        className="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
        onBlur={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        role="button"
        tabIndex={0}
      >
        <div
          className="MuiListItemText-root"
        >
          <span
            className="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
          >
            Native Speakers
          </span>
        </div>
      </div>
    </a>
    <a
      href="/about/subject-areas/"
    >
      <div
        aria-disabled={false}
        className="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
        onBlur={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        role="button"
        tabIndex={0}
      >
        <div
          className="MuiListItemText-root"
        >
          <span
            className="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
          >
            Subject Areas
          </span>
        </div>
      </div>
    </a>
    <a
      href="/lang/english/"
    >
      <div
        aria-disabled={false}
        className="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
        onBlur={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        role="button"
        tabIndex={0}
      >
        <div
          className="MuiListItemText-root"
        >
          <span
            className="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
          >
            Language
          </span>
        </div>
      </div>
    </a>
  </ul>
  <section
    className="sc-gtssRu xXxNn"
  >
    <p>
      cert
    </p>
    <a
      href="/about/team"
    >
      <div
        className=" gatsby-image-wrapper"
        style={
          Object {
            "display": "inline-block",
            "height": 145,
            "overflow": "hidden",
            "position": "relative",
            "width": 100,
          }
        }
      >
        <img
          alt=""
          aria-hidden={true}
          src="data:image/jpeg;base64,/9j/2wBDABALDA4MChAODQ4SERATGCgaGBYWGDEjJR0oOjM9PDkzODdASFxOQERXRTc4UG1RV19iZ2hnPk1xeXBkeFxlZ2P/2wBDARESEhgVGC8aGi9jQjhCY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2P/wgARCAAdABQDASIAAhEBAxEB/8QAGQAAAgMBAAAAAAAAAAAAAAAAAAMBAgUE/8QAFgEBAQEAAAAAAAAAAAAAAAAAAgQD/9oADAMBAAIQAxAAAAFy1Vpn1xRPvz014ZSOM1//xAAgEAABAwIHAAAAAAAAAAAAAAACAAEDBBQQEhMhIjEz/9oACAEBAAEFAjqiYrs1EbygTDm4ql8jpSc7QlCGmD74dr//xAAaEQACAgMAAAAAAAAAAAAAAAAAAQIQERIx/9oACAEDAQE/Acsjw0Vf/8QAGxEAAgEFAAAAAAAAAAAAAAAAAAIBEhMhMVH/2gAIAQIBAT8BpXg2y4xOT//EAB0QAAIBBAMAAAAAAAAAAAAAAAABEgIRITEgMoH/2gAIAQEABj8CassM0iTK77kx6PRuSyzsiL4f/8QAHxABAAIBBAMBAAAAAAAAAAAAAQARITFBUXEQYdHx/9oACAEBAAE/ITvWhvAvsysgN1iNxPYlHDLva5x8pqOB0Z+KxDQW7xMl14XpP//aAAwDAQACAAMAAAAQJ8AP/8QAFxEBAQEBAAAAAAAAAAAAAAAAAQARMf/aAAgBAwEBPxB37LQsvBhhf//EABYRAQEBAAAAAAAAAAAAAAAAAAEAEf/aAAgBAgEBPxAwhxhBS1rf/8QAIBABAQACAQMFAAAAAAAAAAAAAREAIVFBYdFxgaHB8P/aAAgBAQABPxCf0hELGc4sNZz5MOXJFzWKjGlZ1Tv64r270NHkmQmIaz3wliADrreM/V/WV4pQQi98RpDo6G/GEBIb7HjCRCAkz//Z"
          style={
            Object {
              "height": "100%",
              "left": 0,
              "objectFit": "cover",
              "objectPosition": "center",
              "opacity": 1,
              "position": "absolute",
              "top": 0,
              "transitionDelay": "500ms",
              "width": "100%",
            }
          }
        />
        <picture>
          <source
            srcSet="/static/00f598e2bfece3cac12348633027a854/1b3ab/BDUE-Mitglied-Logo.jpg 1x"
          />
          <img
            alt="A blue BDÜ certification badge"
            height={145}
            loading="lazy"
            onLoad={[Function]}
            src="/static/00f598e2bfece3cac12348633027a854/1b3ab/BDUE-Mitglied-Logo.jpg"
            srcSet="/static/00f598e2bfece3cac12348633027a854/1b3ab/BDUE-Mitglied-Logo.jpg 1x"
            style={
              Object {
                "height": "100%",
                "left": 0,
                "objectFit": "cover",
                "objectPosition": "center",
                "opacity": 0,
                "position": "absolute",
                "top": 0,
                "transition": "opacity 500ms",
                "width": "100%",
              }
            }
            width={100}
          />
        </picture>
        <noscript
          dangerouslySetInnerHTML={
            Object {
              "__html": "<picture><source srcset=\\"/static/00f598e2bfece3cac12348633027a854/1b3ab/BDUE-Mitglied-Logo.jpg 1x\\" /><img loading=\\"lazy\\" width=\\"100\\" height=\\"145\\" srcset=\\"/static/00f598e2bfece3cac12348633027a854/1b3ab/BDUE-Mitglied-Logo.jpg 1x\\" src=\\"/static/00f598e2bfece3cac12348633027a854/1b3ab/BDUE-Mitglied-Logo.jpg\\" alt=\\"A blue BDÜ certification badge\\" style=\\"position:absolute;top:0;left:0;opacity:1;width:100%;height:100%;object-fit:cover;object-position:center\\"/></picture>",
            }
          }
        />
      </div>
    </a>
  </section>
  <section
    className="sc-dlnjPT fEVegM"
  >
    <p>
      © 
      2020
       Eisenmann Uebersetzungen
    </p>
    <p>
      gatsby
       
       
       
      <a
        href="https://www.gatsbyjs.org"
      >
        Gatsby
      </a>
    </p>
    <a
      href=""
    >
      Data Privacy
    </a>
  </section>
</footer>
`;
